core/lib/libtomcrypt/gcm.c

```diff
@@ -7,6 +7,7 @@
 #include <crypto/crypto.h>
 #include <crypto/crypto_impl.h>
 #include <stdlib.h>
+#include <string_ext.h>
 #include <string.h>
```

Comment:
nitpicking: swap the 2 lines above for consistency of ordering, since right above we have:
```c
#include <crypto/crypto.h>
#include <crypto/crypto_impl.h>

Reply:
I've sorted the includes with `sort -n` and that's the order it gave. I guess it's ignoring the `_` and `.` so `i` comes after `h`, but `e` before `h`.

Reply:
Ok :). discard my comment.

Reply:
When I apply `shift+V<select lines>:!sort -n` in `vim` I get `string.h` before `string_ext.h`... could this be due to the locale? That looks a bit better IMO because we tend to split things visually at the dot `.`, but it's no big deal.